packages/ui/src/composites/wui-list-select/index.ts

```diff
@@ -0,0 +1,63 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { globalStyles } from '../../utils/ThemeUtil'
+import '../../components/wui-text'
+import '../wui-wallet-image'
+import '../wui-all-wallets-image'
+import styles from './styles'
+import type { IWalletImage } from '../../utils/TypesUtil'
+
+@customElement('wui-list-select')
+export class WuiListSelect extends LitElement {
+  public static styles = [globalStyles, styles]
+
+  // -- state & properties ------------------------------------------- //
+  @property({ type: Array }) public walletImages?: IWalletImage[] = []
+
+  @property() public imageSrc? = ''
+
+  @property() public listName = ''
+
+  @property() public listStatus?: string
```

Comment:
same as above, can be just `status`. As we only have a handful of statuses, perhaps we can make this more strict instead of `string` i.e specific status like installed | recent etc.